social-network/src/main/java/com/levi9/socialnetwork/Model/EUserVerifiedRegistration.java

```diff
@@ -0,0 +1,6 @@
+package com.levi9.socialnetwork.Model;
+
+public enum EUserVerifiedRegistration {
```

Comment:
Naming is the biggest problem of programmers :) 
I would name it something like UserVerificationStatus, no need for E

Reply:
I agree, that is much better and more understandable. I updated the name.